packages/pcd-collection/src/PCDCollection.ts

```diff
@@ -453,6 +453,38 @@ export class PCDCollection {
 
     return collection;
   }
+
+  /**
+   * Merges another PCD collection into this one.
+   * There are two options:
+   * - `filter` is a function used to filter out PCDs from the other
+   *   collection during merging, e.g. to filter out duplicates or PCDs of
+   *   a type that should not be copied.
+   * - `setFolders` is a boolean controlling whether or not the PCDs from
+   *   the other collection should be added to the same folders in this
+   *   collection.
```

Comment:
Document what happens if this is set to false.  Also do you foresee a use for this option as false?  I wouldn't add it until you know it's necessary.